src/AutoRest.CSharp/MgmtTest/Generation/ResourceTestWriter.cs

```diff
@@ -34,6 +33,7 @@ internal class ResourceTestWriter : MgmtBaseTestWriter
         public ResourceTestWriter(CodeWriter writer, Resource resource, BuildContext<MgmtOutputLibrary> context): base(writer, resource, context)
         {
             _resource = resource;
+            _getAllOperation = _resource.AllOperations;
```

Comment:
Shall we rename this variable? `resource.AllOperations` returns a collection that contains all the operations in this resource, and `getAllOperation` stands for one single operation with the name of `GetAll`.